### PR TITLE
FAQ - update 'other_platforms' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Mention CCTG

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -484,7 +484,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "It's not foreseeable at the moment if and when the app will be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification APIs. The community is invited to contribute and use the app for other APIs."
+                            "It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework. The community is invited to contribute and use the app for other frameworks.",
```
Comment (Ein-Tim) [inline src/data/faq.json:487]: Could we please change "The app will use the Apple and Google Exposure Notification Framework."  to something which reflects that the app is already released?

File: src/data/faq_de.json [hunk 2]
```diff
@@ -485,7 +485,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "Es ist im Moment nicht absehbar, wann und ob wir die App in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification APIs verwenden. Die Community kann sich gerne einbringen und die App für andere APIs übernehmen."
+                            "Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden. Die Community kann sich gerne einbringen und die App für andere Frameworks übernehmen.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:488]: Same as in English

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq.json:487]: Suggest changing the historical statement from:

"It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework."

to:

"The Corona-Warn-App uses the Apple and Google Exposure Notification Framework and the Robert Koch Institute (RKI) has correspondingly released the app to the Apple and Google app stores. The RKI does not currently plan to release  to other app stores."

File: src/data/faq.json [hunk 3]
```diff
@@ -484,7 +484,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "It's not foreseeable at the moment if and when the app will be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification APIs. The community is invited to contribute and use the app for other APIs."
+                            "It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework. The community is invited to contribute and use the app for other frameworks.",
+                            "For F-Droid, there exists an inofficial fork of the app under the name <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a> which is maintained by the community. Because the development of the fork is not supervised by the Robert Koch-Institut, we can give no official support for problems with the app published on F-Droid."
```
Comment (MikeMcC399) [inline src/data/faq.json:488]: Change "Robert Koch-Institut" to English name "Robert Koch Institute" or use abbreviation "RKI" from above suggestion.

File: src/data/faq_de.json (hunk 2 above)
Comment (MikeMcC399) [reply src/data/faq_de.json:488]: Suggest changing the historical statement from:

"Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden."

to:

"The Corona-Warn-App verwendet die Apple and Google Exposure Notification Framework, dementsprechend hat das Robert Koch-Institut (RKI) die App in den Apple und Google App Stores veröffentlicht. Derzeit hat das RKI keine Pläne die App in anderen Stores zur Verfügung zu stellen."

File: src/data/faq_de.json [hunk 4]
```diff
@@ -485,7 +485,8 @@
...
                         "textblock": [
-                            "Es ist im Moment nicht absehbar, wann und ob wir die App in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification APIs verwenden. Die Community kann sich gerne einbringen und die App für andere APIs übernehmen."
+                            "Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden. Die Community kann sich gerne einbringen und die App für andere Frameworks übernehmen.",
+                            "Für den F-Droid Store existiert ein inoffizieller Fork der App unter dem Namen <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a>, welcher von der Community betreut wird. Da die Entwicklung des Forks nicht vom Robert-Koch-Institut betreut wird, erhält die im F-Droid Store veröffentlichte App jedoch keinen offiziellen Support."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:489]: "Robert-Koch-Institut" => "Robert Koch-Institut" or use RKI abbreviation from previous suggestion.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I followed on from @Ein-Tim's suggestion and entered some text to reflect that time has moved on and the app has been released.

I also noted about the naming of RKI in English (Robert Koch Institute) and German (Robert Koch-Institut).

File: src/data/faq_de.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq_de.json:488]: @MikeMcC399 The suggestion sounds good, but in the text here there's a mistake, it should be "Die Corona-Warn-App" and not "The Corona-Warn-App".

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed (+ sentence rewording)

@MikeMcC399 I didn't use your recommendation for the reasons that F-Droid support has been neither confirmed nor denied, so the original statement is still valid.

File: src/data/faq_de.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq_de.json]: see above

File: src/data/faq_de.json (hunk 4 above)
Comment (heinezen) [reply src/data/faq_de.json:489]: fixed

**Review by Ein-Tim — APPROVED** [review]
Looks good to me now!

File: src/data/faq.json (hunk 3 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

File: src/data/faq_de.json (hunk 4 above)
Comment (heinezen) [reply src/data/faq_de.json:489]: Actually not fixed, so I have to correct myself. The spelling with one hyphen is the correct one, see https://www.rki.de/DE/Content/Institut/institut_node.html

They changed it in 1994 for whatever reason

**Comment by MikeMcC399** [comment]
@dsarkar 

Same comment here. Suggest to set to draft or close this PR.

**Comment by Ein-Tim** [comment]
This PR is heavily out of sync with the `main` branch, which would require major manual work to make this PR ready to merge.
I therefore request this PR to be closed & if the changes should be applied, this should be done via a new PR.

**Comment by brianebeling** [comment]
> This PR is heavily out of sync with the `main` branch, which would require major manual work to make this PR ready to merge. I therefore request this PR to be closed & if the changes should be applied, this should be done via a new PR.

As Tim already pointed out, at this point a new PR and/or Issue would be more suitable, if this change is planned. Thank you all, still, for the work you put into this change!

**Pull request closed without merging**